code/modules/mob/living/carbon/human/species_types/psyphoza.dm

```diff
@@ -392,6 +455,26 @@ GLOBAL_LIST_INIT(psychic_sense_blacklist, typecacheof(list(/turf/open, /obj/mach
 /obj/effect/psyphoza_spores/proc/upgrade()
 	upgrades = min(upgrade_limit, upgrades+1)
 
+//Keybind for sense
+/datum/keybinding/mob/psychic_sense
```

Comment:
put this in the keybind file please, and since the signal is generically named "species action primary" please make the name and description reflect this